src/viking.js

```diff
@@ -1,11 +1,120 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength;
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+
+  //Viking has access to all of "Soldier"'s methods, including "attack()" via "extends"
+
+  //Damage
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health >= 1) {
+      return `${this.name} has received ${damage} points of damage`;
+    } else {
+      return `${this.name} has died in act of combat`;
+    }
+  }
+  //BattleCry RAAAAAAA!
+  battleCry() {
+    return "Odin Owns You All!";
+  }
+}
 
-// Saxon
-class Saxon {}
+//Saxon
+class Saxon extends Soldier {
+  constructor(health, strength) {
+    super(health, strength);
+  }
+
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health >= 1) {
+      return `A Saxon has received ${damage} points of damage`;
+    } else {
+      return `A Saxon has died in combat`;
+    }
+  }
+}
 
 // War
-class War {}
+class War {
+  constructor() {
+    this.vikingArmy = [];
+    this.saxonArmy = [];
+  }
+
+  addViking(Viking) {
```

Comment:
while this works, be careful with the capital letters, since this now "shadows" the name of the Viking class from the same file! also remember, only classes should be named using PascalCase where here in line 64 Viking is an instance of a class not a class itself! so another reason to name it viking instead :)